pyccel/codegen/printing/pycode.py

```diff
@@ -394,7 +373,7 @@ def _print_Program(self, expr):
         module = modules[0]
         imports = ''.join(self._print(i) for i in expr.imports if i.source_module is not module)
         body     = self._print(expr.body)
-        imports += ''.join(self._print(i) for i in self.get_additional_imports())
+        imports += ''.join(self._print(i) for i in self._additional_imports.values())
```

Comment:
does this private attribute need a getter?

Reply:
Only if it is used outside the class

Reply:
(pylint complains if you use private members. But you can always use private members of your own class otherwise there's no point saving anything privately)

Reply:
got it!